webapp-django/crashstats/settings/bundles.py

```diff
@@ -430,22 +433,22 @@
         for asset_file in v['source_filenames']:
             if asset_file in _used:
                 # Consider using warnings.warn here instead
-                print '{:<52} in {:<20} already in {}'.format(
+                print('{:<52} in {:<20} already in {}'.format(
                     asset_file,
                     k,
                     _used[asset_file]
-                )
+                ))
                 _trouble.add(asset_file)
             _used[asset_file] = k
 
     for asset_file in _trouble:
-        print "REPEATED", asset_file
+        print("REPEATED", asset_file)
```

Comment:
Note that when you use `from __future__ import print_function` these two become identical:
```python
print "string1", "string2"
```
...and...
```python
from __future__ import print_function
print("string1", "string2")
```
In both cases, Python spits out out as `string1 string2`.